core/ecschema-metadata/src/Metadata/OverrideFormat.ts

```diff
@@ -108,7 +109,37 @@ export class OverrideFormat {
   public static isOverrideFormat(object: any): object is OverrideFormat {
     const overrideFormat = object as OverrideFormat;
 
-    return overrideFormat !== undefined && overrideFormat.name !== undefined && overrideFormat.parent !== undefined &&
-             overrideFormat.parent.schemaItemType === SchemaItemType.Format;
+    return overrideFormat !== undefined && overrideFormat.name !== undefined && overrideFormat.parent !== undefined && overrideFormat.parent.schemaItemType === SchemaItemType.Format;
+  }
+
+  /**
+   * Returns a JSON object that contains the specification for OverrideFormat.
+   */
+  public toJSON(): SchemaItemFormatProps {
+    const formatJson = this.parent.toJSON() as any;
```

Comment:
Why `any` here and below?

Reply:
The parent FormatProps has all readonly members.
Without any, I wasn't able to call their toJSON()s and overwrite the JSON object with the OverrideFormat's members.

Reply:
Skipping questions about *why* all the interfaces in this package declare their properties as `readonly`: consider using `as Mutable<FormatProps>` instead. This way you don't lose type safety.
Less efficient alternative would involve spread operator.